torch/_inductor/codegen/cpp.py

```diff
@@ -1229,15 +1229,41 @@ def could_vec(self, name: str, index: sympy.Expr):
             most_inner_var, index
         )
 
+    def is_mask(self, name: str, users: Dict[torch.fx.Node, None]):
+        load_type = V.graph.get_dtype(name)
+        if load_type == torch.bool:
+            user_nodes = set([_node.target for _node in users.keys()])
+            return user_nodes == set(["where", "masked"])
```

Comment:
Do you intend to check if any of the user is either "where" or "masked"? If so, `all(user.target in ("where", "masked") for user in users.keys())` sounds correct then.